interfaces_test.go

```diff
@@ -146,14 +146,49 @@ func TestEventMarshalJSON(t *testing.T) {
 	}}
 	event.StartTime = time.Unix(7, 0).UTC()
 	event.Timestamp = time.Unix(14, 0).UTC()
+	event.DebugMeta = &DebugMeta{
+		SdkInfo: &DebugMetaSdkInfo{
+			SdkName:           "test",
+			VersionMajor:      1,
+			VersionMinor:      2,
+			VersionPatchlevel: 3,
+		},
+		Images: []DebugMetaImage{
+			{
+				Type:      "macho",
+				DebugID:   "12345",
+				CodeFile:  "foo.dylib",
+				ImageAddr: "0xabcd0000",
+			},
+		},
+	}
```

Comment:
Please move your additions to a new test (a new function that starts with "Test", e.g. `TestMarshalEventWithDebugMeta`), and preferably include all newly added fields there (e.g. with two images, "macho" and "proguard", to cover all the fields in DebugMetaImage).
The preamble (`event := NewEvent()`) will stay the same in the new test, but everything else will be purely about DebugMeta testing.